### PR TITLE
Added `linode` and `lin` alias

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Added `linode` and `lin` alias for the linode-cli. Users can now call all existing `linode-cli` commands using the shorthand alias.

## ✔️ How to Test

`make testunit`

### CODE REVIEW

**Review by jriddle-linode — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -83,9 +83,11 @@ def bake_version(v):
...
             "linode-cli = linodecli:main",
+            "linode = linodecli:main",
+            "lin  = linodecli:main",
         ]
     },
     data_files=get_baked_files(),
     python_requires=">=3.6",
     include_package_data=True,
-)
+)
```
Comment (zliang-akamai) [inline setup.py:93]: Can you add a blank line at the end of the file? To get rid of GitHub warning.

**Review by zliang-akamai — APPROVED** [review]

File: lin.sh [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# This is a generated file by Linode-CLI! Do not modify!
```
Comment (lgarber-akamai) [inline lin.sh:1]: You might need to add `lin.sh` and `linode.sh` to `.gitignore` 👍

File: linodecli/completion.py [hunk 3]
```diff
@@ -47,11 +51,17 @@ def get_fish_completions(ops):
     completion_template = Template(
         """# This is a generated file by Linode-CLI! Do not modify!
 complete -c linode-cli -n "not __fish_seen_subcommand_from $subcommands" -x -a '$subcommands --help'
+complete -c linode -n "not __fish_seen_subcommand_from $subcommands" -x -a '$subcommands --help'
```
Comment (lgarber-akamai) [inline linodecli/completion.py:54]: The corresponding unit tests may need to be adjusted to expect these as a part of the output

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]
Great work so far!

**Review by lgarber-akamai — APPROVED** [review]
Great work!

**Pull request merged**